noir_stdlib/src/eddsa.nr

```diff
@@ -1,25 +1,8 @@
 use crate::hash::poseidon;
 use crate::ec::consts::te::baby_jubjub;
 use crate::ec::tecurve::affine::Point as TEPoint;
-// Returns true if x is less than y
-fn lt_bytes32(x: Field, y: Field) -> bool {
-    let x_bytes = x.to_le_bytes(32);
-    let y_bytes = y.to_le_bytes(32);
-    let mut x_is_lt = false;
-    let mut done = false;
-    for i in 0..32 {
-        if (!done) {
-            let x_byte = x_bytes[31 - i] as u8;
-            let y_byte = y_bytes[31 - i] as u8;
-            let bytes_match = x_byte == y_byte;
-            if !bytes_match {
-                x_is_lt = x_byte < y_byte;
-                done = true;
-            }
-        }
-    }
-    x_is_lt
-}
+use crate::field::bn254::lt as field_lt;
```

Comment:
I think it would be good to maintain this as a fallback for when using a non-bn254 curve. We can expose a single `field_lt` function and select an implementation using compile time `if-elses`

Reply:
Did that! Removed the field(bn254) attributes though since those functions would dissapear if the field was non-bn254, leaving those functions non-existing